js-exercises/last-index-of/lastIndexOf.js

```diff
@@ -0,0 +1,18 @@
+
+function lastIndexOf(...args) {
+  const numberToBeSearched = args[0]
```

Comment:
this can be elementToBeSearched / itemToBeSearched because of it might be number/srtring